features/steps/payment_steps.py

```diff
@@ -0,0 +1,14 @@
+from behave import *
```

Comment:
Best not to import everything from a namespace if you can help it.

Maybe you can just do `from behave import when, then`?